Server/JokesOnYou.Web.Api/Controllers/AuthController.cs

```diff
@@ -7,16 +10,53 @@
 
 namespace JokesOnYou.Web.Api.Controllers
 {
-    public class AuthController
+    [Route("api/[controller]")]
+    [ApiController]
+    public class AuthController : ControllerBase
     {
-        public ActionResult Login([FromBody] UserLoginDTO userLoginDto)
+        private readonly IAuthService _authService;
+
+        public AuthController(IAuthService authService)
         {
-            throw new NotImplementedException();
+            _authService = authService;
         }
 
-        public Task<ActionResult> Register([FromBody] UserRegisterDTO userRegisterDto)
+        [AllowAnonymous]
+        [HttpPost("login")]
+        public async Task<IActionResult> Login(UserLoginDTO userLoginDto)
         {
-            throw new NotImplementedException();
+            try
+            {
+                var user = await _authService.LoginAsync(userLoginDto);
+                return Ok(user);
+            }
+            catch (UserLoginException ex)
+            {
+                return BadRequest(ex.Message);
```

Comment:
return Unauthorized

Reply:
Oké so 401 it is ;)

`**400 Bad Request**`
The server cannot or will not process the request due to an apparent client error (e.g., malformed request syntax, size too large, invalid request message framing, or deceptive request routing).
`**401 Unauthorized**`
Similar to 403 Forbidden, but specifically for use when authentication is required and has failed or has not yet been provided.

It seems to be the standard.

source: https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#4xx_client_errors